webapp/javascript/components/Settings/Apps/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+import { useAppDispatch, useAppSelector } from '@webapp/redux/hooks';
+import {
+  selectApps,
+  reloadApps,
+  deleteApp,
+} from '@webapp/redux/reducers/settings';
+import { addNotification } from '@webapp/redux/reducers/notifications';
+import { type App } from '@webapp/models/app';
+import Input from '@webapp/ui/Input';
+import TableUI from '@webapp/ui/Table';
+import cl from 'classnames';
+
+import appsStyles from './Apps.module.css';
+import tableStyles from '../SettingsTable.module.scss';
+import { getAppTableRows } from './getAppTableRows';
+
+const headRow = [
+  { name: '', label: '', sortable: 0 },
```

Comment:
I think this one is not needed. E.g look at this screenshot, "Name" is too far to the right.
![Screen Shot 2022-08-19 at 3 00 57 AM](https://user-images.githubusercontent.com/662636/185595355-85d3e2f9-7fde-4545-8c05-9dc27103ae7e.png)

It looks like there are 2 `td` elements per `tr`, so it would make sense to have 2 for `th` as well.

Reply:
Addressed in https://github.com/pyroscope-io/pyroscope/pull/1424/commits/66cc28df6dc973c3295e501e65807ebd888b5195